addons/web/static/src/views/calendar/hooks/square_selection_hook.js

```diff
@@ -42,15 +42,18 @@ function getResult(ctx) {
         }
     }
     const selector = selectorParts.join(",");
-    return { selectedCells: [...ref.el.querySelectorAll(selector)] };
+    return { selectedCells: [...ref.el.querySelectorAll(selector)].filter(ctx.cellIsSelectable) };
```

Comment:
Could this crash if cellIsSelectable is not defined? It seems that it is only used when it is, inside useBlockSelection, but double checking :)

Reply:
The only place where `useBlockSelection` is called is inside `useSquareSelection` and we have this line which should fallback if it's not specified :slightly_smiling_face: : 
` const cellIsSelectable = params.cellIsSelectable || (() => true);`